react/react-state/react-state_box/src/App.js

```diff
@@ -1,18 +1,22 @@
+import { useState } from "react";
 import "./styles.css";
 
 export default function App() {
-  let isActive = false;
+  const [isActive, setIsActive] = useState(false);
 
   function handleClick() {
-    isActive = !isActive;
-    // Check that the value changes correctly.
+    setIsActive(!isActive);
     console.log(isActive);
   }
 
   return (
     <main>
       <div className={`box ${isActive ? "box--active" : ""}`} />
-      <button onClick={handleClick}>Activate</button>
+      {isActive ? (
+        <button onClick={handleClick}>Deactivate</button>
```

Comment:
You can also put the ternary operator inside the button text, i.e. <button...>{isActive ? "Deactivate" : "Activate"}</button>